src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,172 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.line.exception.DeleteOnlySectionException;
+import nextstep.subway.line.exception.DuplicatedSectionException;
+import nextstep.subway.line.exception.NoneMatchSectionException;
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Embeddable
+public class Sections {
+    public static final int DIFFERENCE_SECTIONS_STATIONS_SIZE = 1;
+    public static final int CANNOT_DELETE_SIZE = 1;
+
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    private List<Section> sections = new ArrayList<>();
+
+    public void add(Section section) {
+        this.sections.add(section);
+    }
+
+    public boolean isEmpty() {
+        return sections.isEmpty();
+    }
+
+    public int size() {
+        return sections.size();
+    }
+
+    public List<Station> getStations() {
+        if (handleEmptyList(sections.isEmpty())) return Arrays.asList();
```

Comment:
기본적으로 수행하는 `line` 이 한줄이어도 `if`, `for` 문 등에서는 `대괄호{}` 를 포함하는것이 컨벤션으로 알고있습니다 :) 
```suggestion
        if (handleEmptyList(sections.isEmpty())) {
                return Arrays.asList();
        }
```